### ISSUE TITLE
The grammar for QNames is not followed

### ISSUE THREAD

**Issue by rracariu** [Novo-Int/xeto-vscode-extension#36]
According to this https://github.com/haxall/haxall/blob/main/src/core/xeto/grammar.txt#LL34C3-L34C60

The following is valid:
```
Type: {
  el: sys::Dict
}
```

The LSP parser errors out with:
```
Expecting end of proto: comma or newline, not :
```

**Comment by hades2510** [comment]
should be fixed in `0.1.25`

**Comment by rracariu** [comment]
verified with 0.1.26
